brightnessctl: add page (#6516)
tldr-pages/tldr: pages/linux/brightnessctl.md

# brightnessctl

> Utility for reading and controlling device brightness for GNU/Linux operating systems.

> > For more information about brightnessctl development, visit: https://github.com/Hummer12007/brightnessctl.

Reviewer: We use this standard:
```suggestion
> More information: <https://github.com/Hummer12007/brightnessctl>.
```

> - List devices with available brightness control:

Reviewer 2: Perhaps this will read better?
```suggestion
- List devices with an available brightness control:
```

Reviewer 3: I don't think it's neseccary.

Author: Me neither. Though tt could very well be 
`- List devices with available brightness controls:`
 considering the fact that there could be multiple devices with multiple controls.

Reviewer 3: I think both versions are correct and I don't really prefer any of them.

Reviewer 4: I don't like the usage of available

Maybe this fits better?

```suggestion
- List devices with changeable brightness:
```

`brightnessctl -l`

> - Get the currently set brightness of the device:

Reviewer 3: `The device` isn't really specific
```suggestion
- Get the currently set brightness of the display backlight:
```

> - Get the currently set brightness of the display backlight:

Reviewer 4: ```suggestion
- Print the currentl brightness of the display backlight:
```

`brightnessctl get`

> - Set brightness to certain percent of the maximum (eg. 50%):

Reviewer 3: ```suggestion
- Set the brightness of the display backlight to certain percent of the maximum:
```
I think we could also remove the last part, because you kind of already expect that behaviour
```suggestion
- Set the brightness of the display backlight:
```

> - Set the brightness of the display backlight to certain percent of the maximum:

Reviewer 2: > 
Perhaps this will read better?
```suggestion
- Set the brightness of the display backlight to a percenage of the maximum supported value:
```

Reviewer 3: As I said [here](https://github.com/tldr-pages/tldr/pull/6516#discussion_r708198885), I think we can shorten that down to
```suggestion
- Set the brightness of the display backlight:
```

Because brighness is almost always given in percent, so it's the behaviour you'd expect anyways.

> - Set the brightness of the display backlight to a percenage of the maximum supported value:

Reviewer 4: ```suggestion
- Set the brightness of the display backlight to maximum percentage:
```

> `brightnessctl set 50%`

Reviewer: We use this token syntax for argument options or user values:
```suggestion
`brightnessctl set {{50%}}`
```

> - Increase brightness by certain percent:

Reviewer 4: ```suggestion
- Increase brightness by a specified increment:
```

> - Increase brightness by certain percent (eg. 10%):

Reviewer 5: I think "10%" is clearly shown in the example, no need to mention in description.
```suggestion
- Increase brightness by certain percent:
```

> `brightnessctl set +10%`

Reviewer: ```suggestion
`brightnessctl set {{+10%}}`
```

> - Decrease brightness by certain percent:

Reviewer 4: ```suggestion
- Decrease brightness by a specified decrement:
```

Author: I do not like 'decrease and 'decrement' in the same sentence, maybe the following is better:
`- Decrease brightness by a specified amount:`

Reviewer 4: I don't personally see the issue, since ` - Increase __________ increment` was just applied, which in return is just the complete opposite of each other completely, since the words chosen are antonyms

> - Increase brightness by certain percent (eg. 10%):
> 
> `brightnessctl set {{+10%}}`
> 
> - Decrease brightness by certain percent (eg. 10%):
> 
> `brightnessctl set {{-10%}}`

Reviewer 3: What do you think about this?
```suggestion
- Increase or decrease the brightness of the display backlight by a certain percentage:

`brightnessctl set {{+|-10%}}`
```

Reviewer: I'm not sure, splitting the `+` and `-` is easier to read; and this page has 5 examples. :ok_hand:

> - Decrease brightness by certain percent (eg. 10%):

Reviewer 5: ```suggestion
- Decrease brightness by certain percent:
```

> `brightnessctl set -10%`

Reviewer: ```suggestion
`brightnessctl set {{-10%}}`
```
